src/Schema/AbstractSchemaManager.php

```diff
@@ -36,6 +37,22 @@
  */
 abstract class AbstractSchemaManager
 {
+    /**
+     * The key that represents null in an array where key is schema name.
+     */
+    protected const NULL_SCHEMA_KEY = "\x00";
```

Comment:
This could be any value, which isn't a valid SQL identifier. An empty string would look even closer to `null` but I think it would be more error-prone.